source/root/usr/lib/logdata-anomaly-miner/aminer.py

```
@@ -92,10 +91,29 @@ def initialize_loggers(aminer_config, aminer_user, aminer_grp):
     datefmt = '%d/%b/%Y:%H:%M:%S %z'
 
     persistence_dir = aminer_config.config_properties.get(AMinerConfig.KEY_PERSISTENCE_DIR, AMinerConfig.DEFAULT_PERSISTENCE_DIR)
+    log_dir = aminer_config.config_properties.get(AMinerConfig.KEY_LOG_DIR, AMinerConfig.DEFAULT_LOG_DIR)
+    if log_dir == AMinerConfig.DEFAULT_LOG_DIR:
+        try:
+            os.makedirs(log_dir)
+            from pwd import getpwnam
+            from grp import getgrnam
+            os.chown(log_dir,
+                    getpwnam(aminer_user).pw_uid,
+                    getgrnam(aminer_grp).gr_gid)
+        except FileExistsError:
+            pass
```

Please add a short comment that clarifies why ignoring this error is intentional.